### ISSUE TITLE
Presentation - add Session Title, maybe Chair

### ISSUE THREAD

**Issue by adam3smith** [citation-style-language/zotero-bits#60]
https://forums.zotero.org/discussion/30294/additional-fields-needed-for-conference-paper/#Item_7

According to APA, papers presented as part of an organized panel or symposium should be cited as

Wiernik, B. M. (2013, April). Effects of respondent characteristics on organizational employee sustainability survey responses. In Dilchert, S. & Wiernik, B. M. (Chairs), Practical lessons in survey methods for measuring employee green behaviors. Symposium conducted at the annual conference of the Society for Industrial and Organizational Psychology, Houston, Texas.

Getting this 100% right might be hard but I would suggest to
1. Add a field "Session Title" and map it to csl container-title" (that would be the "Practical lessons..." (I'm quite sure this is a good idea)
2. To get the chairs might be harder. My suggestion would be to add a creator type "Chair" in Zotero and map it to the rarely used csl "collection-editor" - that's however still on the hack-ish side. Alternatives would be a new creator type  "Chair" (including csl Change) - a rather major change for such a rare cite-form, or to just map to "editor" and accept that we'll get the wrong label.  (CSL type is "speech")

Edited to clarify Rintze's questions.

**Comment by bwiernik** [comment]
@bdarcus Do you think `chair` could make it into 1.0.2? 

The current version of apa.csl uses `container-title` with `speech` for citations to presentations in a symposium. That seems to work well. It seems reasonable to me to think of a session containing papers as a “container”, and something like a lecture series or a theme track at a conference as a “collection”. 

If we don’t want to use `container-title` for this purpose (and I can see the argument against it), how about `event-container-title` and `event-collection-title` as parallels to `container-title` and `collection-title` for event information?

In considered shorter labels like `session-title` and `track-title`, but I think using the same container/collection vocabulary and explicitly including “event” is clearer.

**Comment by bdarcus** [comment]
We should resolve this now. But to pull back, some propositions:

1. an academic conference is an event; so is a hearing, a speech, a performance
2. events can have sub-events; in a conference, a session would be an example
3. what we're talking about here is roles within the above; with "chair" being a subtype of "organizer"
4. when citing, one often, but not always, cites a document about the event in question; a recording of a performance, a transcript of a hearing, a conference proceeding

Is that all correct?

So how can we best deal with all this in a unified way, while accommodating the specific case?

One step might be to add "organizer" and "chair"?

Titles?

**Comment by bdarcus** [comment]
For conferences, we need a way to distinguish three titles, and potentially associate contributors with each of these levels:

1. paper
2. session
3. conference

Right?

So:

1. paper (`title`)
2. session (`containing-title`)
3. conference (here's where I get stuck)

Maybe we can just change the documentation to make clear that one can use `container-title` for 2?

**Comment by bwiernik** [comment]
**tl;dr: I suggest we add `chair` and `organizer` creator variables, update documentation for `container-title` and `collection-title`**

A few points:
1. Academic conferences and speeches are conceptually similar events with similar citation needs. Performances and hearings often have different citation needs, so those probably need distinct types. Those are discussed in other issues, so let's set them aside for now. 
2. Citing conferences/speeches have two requirements that aren't currently met by CSL:
    1. Unique creator roles
    1. Sub-event structure
3. Whether what is cited is a document/recording or not is not that relevant I don't think. For the specific case of conference presentations, these are generally _not_ to a document, but to the event itself, which may or may not be recorded or retrievable. A document might be available online, but the citation form still follows the event structure unless it's formally published in a proceedings.

## Creator roles

1. `chair` and `organizer` would be good. I think we could maybe get by with just `organizer` and have a `chair` term, but I don't see a problem with having both types of creators. 
1. Other potential conference-related creator roles are "moderator" and "discussant".
    1. I don't think these roles are cited often enough for them to be necessary in CSL (vs. just as variables in a client mapped to `author`) 

## Event level titles

I can see four levels of information

1. paper (`title`)
1. session (`?containing-title?`)
1. series/collection (`?collecting-title?`)
    1. A collection of multiple sessions
    1. Think things like "Meredith Crawford Lecture Series" or "Theme Track on Privacy and Technology"
    1. Analogous to a special issue of a journal or a book series
1. conference (the existing `event` variable)

The question is whether we need new variables for (2) and (3) above or if we can just update documentation for `container-title` and `collection-title` to encompass the above. I think we should just update documentation.

For the question of what level is `organizer` or `chair` associated with, that's basically the same problem as `editor` or `translator` for a `chapter`. I'm not sure there is a way for this to be unambiguous with the current flat data model. Relying on heuristics (e.g., `chair` is the chair of session [`container-title`], `organizer` is organizer of conference [`event`]) should work in the large majority of cases.

**Comment by denismaier** [comment]
> For the question of what level is `organizer` or `chair` associated with, that's basically the same problem as `editor` or `translator` for a `chapter`. I'm not sure there is a way for this to be unambiguous with the current flat data model. Relying on heuristics (e.g., `chair` is the chair of session [`container-title`], `organizer` is organizer of conference [`event`]) should work in the large majority of cases.

So, as a sidenote, you also see the need for a revised datamodel? (as outlined `https://github.com/citation-style-language/csl-evolution/issues/12`.)

**Comment by bwiernik** [comment]
So, summary for Zotero:
1. Add **Chair** (`chair`) and **Organizer** (`organizer`) creators to Presentation.
1. Add **Session Title** (`container-title`) and **Series** (`collection-title`) fields to Presentation.

**Comment by dstillman** [comment]
So `sessionTitle` will be base-mapped to "Publication", similar to `proceedingsTitle` in `conferencePaper`.

**Comment by bwiernik** [comment]
Yes, exactly.

Also note I had a typo above. Organizer should be mapped to `organizer`.
